os/network-config-with-networkd.md

```diff
@@ -70,8 +70,6 @@ By default, even if you've already set a static IP address and you have a workin
 
 coreos:
   units:
-    - name: systemd-networkd.service
-      command: stop
```

Comment:
Yes, I've checked that several times. Sometimes node's network is up, sometimes - not.